recipes/basemap/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+    name: basemap
+    version: 1.0.8.dev
+
+source:
+  git_url: https://github.com/matplotlib/basemap.git
+  git_tag: 116aefb
+  patches:
+      - basemap-1.0.8dev-datadir.patch
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - python
+        - numpy x.x
+        - geos >=3.4.2
+    run:
+        - python
+        - numpy x.x
+        - geos >=3.4.2
+        - matplotlib
+        - pyproj
+        - pyshp
+
+test:
+    imports:
+        - mpl_toolkits.basemap
+
+about:
+    home: http://matplotlib.org/basemap
+    license: GNU GPL
```

Comment:
There are information on all the licenses for the dependencies and the data, but basemap states OSI Approved only :unamused: 

(See: https://github.com/matplotlib/basemap/search?utf8=%E2%9C%93&q=license)